plugins/toolbox/config.d.ts

```diff
@@ -0,0 +1,60 @@
+import type {
+  CornerDotType,
+  CornerSquareType,
+  DotType,
+  ShapeType,
+} from 'qr-code-styling';
+
+export interface Config {
+  app: {
+    toolbox: {
```

Comment:
Can you make these optional and the use some sensible defaults in the tool? Otherwise people who take this in to use will face problems as the configuration is missing.